visbrain/io/read_sleep.py

```diff
@@ -195,10 +196,14 @@ def __init__(self, data, channels, sf, hypno, href, preload, use_mne,
                 hypno = np.zeros((npts,), dtype=np.float32)
 
         # ---------- SCALING ----------
-        # Check amplitude of the data and if necessary apply re-scaling
-        if np.abs(np.ptp(data, 0).mean()) < 0.1:
-            warn("Wrong data amplitude for Sleep software.")
-            data *= 1e6
+        # Assume that the inter-quartile amplitude of EEG data is ~50 uV
+        iqr_data = iqr(data, axis=1)
+
+        for idx_chan, iqr_chan in enumerate(iqr_data):
+            if iqr_chan < 1:
+                mult_fact = np.floor(np.log10(50 / iqr_chan))
+                warn("Wrong channel data amplitude. Multiplying data amplitude by 10^%i" % mult_fact)
```

Comment:
- Line too long. See : https://travis-ci.org/EtienneCmb/visbrain/jobs/444620569
- `mult_fact` is a float, not int so you can't use `%i`